utils/integration-functions.R

```diff
@@ -107,3 +107,109 @@ add_integrated_pcs <- function(merged_sce,
   return(merged_sce)
   
 }
+
+plot_asw <- function(asw_df,
+                     seed = NULL, 
+                     by_batch, 
+                     batch_label, 
+                     plot_colors = NULL, 
+                     legend_labels = NULL) {
```

Comment:
```suggestion
                     batch_label) {
```
Based on my comment below, remove these.